test/enhance/test_rgb_shift.py

```diff
@@ -47,4 +50,9 @@ def test_rgb_shift(self, device, dtype):
         shifted = kornia.enhance.shift_rgb(image, r_shift, g_shift, b_shift)
         expected = torch.tensor([[[[0.3, 0.1]], [[0.5, 0.7]], [[0.1, 0.4]]]], device=device, dtype=dtype)
 
-        return (shifted == expected).all()
+        utils.assert_close(shifted, expected)
+
+
+if __name__ == "__main__":
```

Comment:
this should be removed: to run this particular test you can use `pytest -k TestRGBShift`

Reply:
Yes, I've removed it, thanks.